packages/fx-core/templates/plugins/resource/bot/bicep/module.template.bicep

```diff
@@ -1,27 +1,31 @@
-module botProvision '{{PluginOutput.fx-resource-bot.Modules.botProvision.Path}}' = {
+module botProvision '\{{PluginOutput.fx-resource-bot.Modules.botProvision.Path}}' = {
   name: 'botProvision'
   params: {
-    botAadClientId: bot_aadClientId
-    botDisplayName: bot_displayName
     botServerfarmsName: bot_serverfarmsName
+    {{#ifCond createNewBotService true}}
     botServiceName: bot_serviceName
+    botAadClientId: bot_aadClientId
+    botDisplayName: bot_displayName
+    {{/ifCond}}
     botServiceSKU: bot_serviceSKU
     botWebAppName: bot_sitesName
     botWebAppSKU: bot_webAppSKU
-    {{#contains 'fx-resource-identity' Plugins}}
-    identityName: {{../PluginOutput.fx-resource-identity.Outputs.identityName}}
-    {{/contains}}
+    \{{#contains 'fx-resource-identity' Plugins}}
```

Comment:
The `\` symbot is used to bypass the handlebars syntax during handlebars compilation. This part of handlebars syntax will be compiled by solution plugin rather than bot resource plugin.

Reply:
You can just generate the final content based on the available plugins, no need to escape it here which makes the template complicated.

Reply:
Yes. I worked. Let me fix it in next PR.